app/models/feature_flag.rb

```diff
@@ -12,6 +12,9 @@ class FeatureFlag
   field :default_value, type: Boolean, default: false
```

Comment:
I think we should add a `validates_uniqueness_of` validation for `name` - this will prevent us from accidentally creating more than one feature flag with the same name.